index.html

```diff
@@ -160,119 +173,126 @@ <h3>Multi-Post Stories <br> Gain+Glory</h3>
+                    <li>API Design</li>
+                </ul>
+            </div>
+        </div>
     </section>
-    <!-- Form-->
-    <section class="contact">
+
+    <section id="contact" class="contact">
         <div class="contactHeading">
             <h3>
                 I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-
             </h3>
         </div>
         <div class="contactForm">
-            <form action="https://formspree.io/f/xayzredb" method="POST">
+            <form action="https://formspree.io/f/moqzgvge" method="POST">
                 <ul>
                     <li>
-                        <input name="fullName" type="text" placeholder="Full Name" maxlength="30" required>
+                        <input name="fullName" type="text" placeholder="Full name" maxlength="30" required>
                     </li>
                     <li>
-                        <input name="user_email" type="email" placeholder="Email address" required>
+                        <input name="email" type="email" placeholder="Email address" required>
                     </li>
                     <li>
-                        <textarea name="user_message" maxlength="500" placeholder="Hey I’d like to connect to..." required></textarea>
+                        <textarea name="messageBox" maxlength="500" cols="40" rows="5"
+                            placeholder="Hey I’d like to connect to..." required> </textarea>
                     </li>
                     <li>
-                        <button type="submit" >Get in Touch</button>
+                        <button type="submit" value="">Get in Touch</button>
                     </li>
                 </ul>
             </form>
-            <p>cheptoopatience81@gmail.com</p>
-        </div> 
-    <div class="socialIcons">
-        <ul>
-            <li>
-                <a href="https://twitter.com/CheptooGyts">
-                    <img src="logo/twitter.png" alt="twitter">
-                </a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/in/cheptoo-patience-73499015a">
-                    <img src="logo/Linkedin icon.png" alt="Linkedin">
-                </a>
-            </li>
-            <li>
-                <a href="#"><img src="logo/medium.png" alt=""></a>
-            </li>
-            <li>
-                <a href="https://github.com/Cheptoo21">
-                    <img src="logo/github.png" alt="github">
-                </a>
-            </li>
-            <li>
-                <a href="#"><img src="logo/angellist.png" alt=""></a>
-            </li>
 
-        </ul>
-    </div>
+        </div>
+    </section>
+    <p class="contactEmail">cheptoopatience81@gmail.com</p>
     <div class="contactBackground">
-        <img src="logo/illustration-contact-form.png" alt=" ">
+        <img class="illustration1" src="logo/illustration-1-contact-form-desktop.svg" alt="Illustration1">
```

Comment:
- [OPTIONAL] This illustration is not showing up and breaking up in the design. kindly consider fixing it. This was missed by the previous reviewer so consider fixing it to make your website professional.